bogenliga/bogenliga-application/src/main/java/de/bogenliga/application/springconfiguration/security/jsonwebtoken/JwtTokenProvider.java

```diff
@@ -180,13 +180,23 @@ private String createToken(final String username, final Long id, final Long vers
         final Date now = new Date();
         final Date validity = new Date(now.getTime() + validityInMilliseconds);
 
+        // if user is default user (not logged in user) expiration time is not set
+        // since the default should be logged in "forever"
+        if(username.equals("ligadefault")) {
+            return Jwts.builder()
+                    .setClaims(claims)
+                    .setIssuedAt(now)
+                    .signWith(SignatureAlgorithm.HS256, secretKey)
+                    .compact();
+        } else {
         return Jwts.builder()
                 .setClaims(claims)
                 .setIssuedAt(now)
                 .setExpiration(validity)
                 .signWith(SignatureAlgorithm.HS256, secretKey)
                 .compact();
     }
```

Comment:
Du kannst die Dinge, die für beide Fälle gemacht werden müssen einmal anfangs machen und dann mit einer if nur die Expiration in dem einen Fall setzen. Dann hast du nicht alles andere doppelt.